ta4j-core/src/main/java/org/ta4j/core/num/Num.java

```diff
@@ -41,6 +41,21 @@
  */
 public interface Num extends Comparable<Num>, Serializable {
 
+    /**
+     * @return the Num of 0
+     */
+    Num zero();
+
+    /**
+     * @return the Num of 1
+     */
+    Num one();
+
+    /**
+     * @return the Num of 100
+     */
+    public Num hundred();
```

Comment:
Please remove `public` modifier from interface.


we could add a `default` implementation to those methods like

```java
default Num hundred(){
    valueOf(100);
}
```

so once is not forced to override this methods and the implementation would still work. Using this defaults we could remove the implementation from `NaN` class